src/event-manager/event-callback-manager.ts

```diff
@@ -0,0 +1,82 @@
+import EventFilter from './event-filter';
+import Subscription from './subscription';
+import { BlockchainEventTypes, EventConfigType } from '../types';
+import { PushId } from '../ain-db/push-id';
+
+export default class EventCallbackManager {
+  private readonly _filters: { [filterId: string]: EventFilter };
+  private readonly _filterIdToSubscriptionId: { [filterId: string]: string };
+  private readonly _subscriptions: { [subscriptionId: string]: Subscription };
+
+  constructor() {
+    this._filters = {};
+    this._filterIdToSubscriptionId = {};
+    this._subscriptions = {};
```

Comment:
could you use new Map() instead for the three member variables above? It's a recommended way.

Reply:
Looks nice! Done